test/stability/testDivide/testDivide.go

```diff
@@ -129,6 +132,13 @@ func executeTest(configFile, target string, testScenario uint, testType uint) er
 		return errors.New("testScenario should be in interval [0, 3]")
 	}
 	gTestType = testType
+	countRes = 0
```

Comment:
move initialization to place where vars declared

Reply:
No. All tests are re-init now. This variables should be zero before each run.